doc/man3/ERR_set_mark.pod

```diff
@@ -2,8 +2,8 @@
 
 =head1 NAME
 
-ERR_set_mark, ERR_clear_last_mark, ERR_pop_to_mark, ERR_count_to_mark - set
-mark, clear mark and pop errors until mark
+ERR_set_mark, ERR_clear_last_mark, ERR_pop_to_mark, ERR_count_to_mark, ERR_pop -
+set mark, clear mark and pop errors until mark
```

Comment:
should the description be expanded to include counting the number of errors to a mark, and pop a single mark?